lib/src/components/Banner.tsx

```diff
@@ -144,15 +110,67 @@ function	Banner({
 				}
 				{variant === 'split' ? <SplitVariant image={image as string} /> : null}
 				{variant === 'background' ? <BackgroundVariant image={image as string} /> : null}
-
-				{canClose ? (
-					<button onClick={onTryToClose} className={'absolute top-4 right-4'}>
-						<IconCross className={`w-6 h-6 cursor-pointer ${variant === 'split' && 'text-surface'}`} />
-					</button>
-				) : null}
 			</div>
 		</div>
 	);
 }
 
-export {Banner};
\ No newline at end of file
+
+export type	TBannerPagination = {
```

Comment:
Adding the BannerPagination. This may not be a good name. The idea of this component is to be able to act on the banner (close or not, with pagination or not)